sixtools/AD_Handler.py

```diff
@@ -0,0 +1,107 @@
+from databroker.assets.handlers_base import HandlerBase
+import os.path
+import pandas as pd
+import numpy as np
+
+
+class HDF5SingleHandler_centroid7xn(HandlerBase):
+    '''Handler for hdf5 data stored 1 image per file by the rixscam xip plugin.
+
+    This will work with all hdf5 files that are a 7xn array where n is the
+    number of centroid 'events' and the 7 'columns' have the format:
+
+        column 1: x
+        column 2: y
+        column 3: x eta correction
+        column 4: y eta correction
+        column 5: y eta correction & isolinear correction
+        column 6: "sum of 3x3 region" (though, this could also mean 2x2 region)
+        column 7: XIP mode (which I assume to be either 3x3 or 2x2)
+
+    This handler will ouput the data as a pandas dataframe with the 'fields'
+    (in order): `x`, `y`, `x_eta`, `y_eta`, `y_eta_+_iso`, `sum regions`,
+    XIP_mode.
+
+    Parameters
+    ----------
+    fpath : string
+        filepath
+    template : string
+        filename template string.
+    filename : string
+        filename
+    key : string
+        the 'path' inside the file to the data set.
+    frame_per_point : float
+        the number of frames per point.
+    '''
+    specs = {'AD_HDF5_SINGLE_C7xn'} | HandlerBase.specs
+
+    def __init__(self, fpath, template, filename, key, frame_per_point=1):
+        self._path = os.path.join(fpath, '')
+        self._fpp = frame_per_point
+        self._template = template
+        self._filename = filename
+        self._key = key
+        self._column_names = ['x', 'y', 'x_eta', 'y_eta', 'y_eta+iso',
```

Comment:
I'm not sure about the 6th column called 'sum_regions'  .   Let me get back to you today about that.

I agree:  'y_eta_iso' is preferable from my side.

Reply:
@ambarb and @stuwilkins I will make the change to `y_eta_iso` on Monday. I was also not sure about `sum_regions` but given that it could be either `sum_3x3_region` or `sum_2x2_region` I didn't have a good better idea.